core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -55,7 +55,7 @@ public SetupWizard(Jenkins j) throws IOException {
             
             HudsonPrivateSecurityRealm securityRealm = new HudsonPrivateSecurityRealm(false, false, null);
             j.setSecurityRealm(securityRealm);
-            String randomUUID = UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH);
+            String randomUUID = System.getProperty("jenkins.install.initialAdminPassword", UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH));
```

Comment:
please can we stop saying installer...  I keep thinking you mean "jenkinsci/packages" installers.  This isn't an installer per say - and the "installers" would not be able to set this flag for a one time use.